code/src/nmea_parser.cpp

```diff
@@ -61,15 +92,35 @@ namespace r2d2::location {
                 break;
             }
             case GGA::altitude_measurement: {
-                location.altitude_measurement = gps_message[i];
+                location.altitude_unit = gps_message[i];
                 break;
             }
             case GGA::geoid_height: {
                 location.geoid_height = atof(gps_message + i, sep_offset);
                 break;
             }
             case GGA::geoid_height_measurement: {
-                location.geoid_height_measurement = gps_message[i];
+                location.geoid_height_unit = gps_message[i];
+                break;
+            }
+            case GGA::diff_age: {
+                // this parameters is not transmitted if we dont have a dgps fix
+                // or higher.
```

Comment:
This doesn't really make sense, what does 'higher than a dgps fix' mean?

Reply:
This refers to the values in the `gga_s` struct. Higher than dgps =  `>= 2`